src/main/java/fr/quatrevieux/araknemu/game/fight/turn/action/move/Move.java

```diff
@@ -56,9 +61,19 @@ public boolean validate() {
 
     @Override
     public ActionResult start() {
-        result = new MoveSuccess(fighter, path);
+        try {
+            for (PathValidatorFight validator : validators) {
+                path = validator.validate(this, path);
+            }
+        } catch (PathValidationException e) {
```

Comment:
Do not use exception as control flow.
Prefer return a `MoveResult` instead, and stop the loop if the result is failed like : 
```suggestion
        result = new MoveSuccess(fighter, path);
        
        for (PathValidatorFight validator : validators) {
                result = validator.validate(this, result);
                
                if (result.failed()) {
                    break;
                }
        }
        
        return result;
```